dtc-lexer.l

```diff
@@ -200,7 +200,7 @@ static void PRINTF(1, 2) lexical_error(const char *fmt, ...);
 			return DT_LABEL_REF;
 		}
 
-<*>"&{/"{PATHCHAR}*\}	{	/* new-style path reference */
+<*>"&{"{PATHCHAR}*\}	{	/* new-style path reference */
```

Comment:
This is strictly broader than the `DT_LABEL_REF` pattern, so it should be possible to collapse both ref types into a single token type.

Reply:
Can't follow you here. `DT_LABEL_REF` is `<*>\&{LABEL}` (unescaped `{}`, e.g. `&label`), while `DT_PATH_REF` is `<*>"&{"{PATHCHAR}*\}` (escaped `{}`, e.g. `&{label/}`).

So no pattern is strictly broader than the other?

Reply:
Oh, sorry, completely missed the `{}`.  Disregard.